src/category_theory/types.lean

```diff
@@ -48,6 +48,11 @@ variables {D : Type u'} [𝒟 : category.{u'} D] (I J : D ⥤ C) (ρ : I ⟶ J)
 
 @[simp] lemma hcomp (x : (I ⋙ F).obj W) : (ρ ◫ σ).app W x = (G.map (ρ.app W)) (σ.app (I.obj W) x) := rfl
 
+@[simp] lemma hom_inv_id_apply (f : X ≅ Y) (x : F.obj X) : F.map f.inv (F.map f.hom x) = x :=
+congr_fun (F.map_iso f).hom_inv_id x
+@[simp] lemma inv_hom_id_apply (f : X ≅ Y) (y : F.obj Y) : F.map f.hom (F.map f.inv y) = y :=
```

Comment:
```suggestion
@[simp] lemma map_hom_map_inv_apply (f : X ≅ Y) (y : F.obj Y) : F.map f.hom (F.map f.inv y) = y :=
```
Feel free to ignore this, and enlighten me about your choice (-;

Reply:
Nope, it was a dumb name.